app/Helpers/render/comment.php

```diff
@@ -113,7 +113,7 @@ function RenderArticleComment(
     $deleteIcon = '';
 
     if ($user && $user === $localUser || $allowDelete) {
-        $img = "<img src='" . asset('assets/images/icon/cross.png') . "' width='16' height='16' alt='delete comment'/>";
+        $img = Blade::render('<x-fas-xmark class="h-5 w-5" style="color: red" />');
```

Comment:
There is no reason to use an inline style here. Please use a Tailwind classname.